lib/src/dartdoc_options.dart

```diff
@@ -432,14 +407,15 @@ abstract class DartdocOption<T> {
   void _validatePaths(_OptionValueWithContext<T> valueWithContext) {
     if (!mustExist) return;
     assert(isDir || isFile);
-    List<String> resolvedPaths;
+    var resolvedPaths = <String>[];
     var value = valueWithContext.value;
     if (value is String) {
       resolvedPaths = [valueWithContext.resolvedValue as String];
     } else if (value is List<String>) {
       resolvedPaths = valueWithContext.resolvedValue as List<String>;
     } else if (value is Map<String, String>) {
-      resolvedPaths = (valueWithContext.resolvedValue as Map).values.toList();
+      resolvedPaths = (valueWithContext.resolvedValue as Map).values.toList()
+          as List<String>;
```

Comment:
I wonder if a spread formats nicer? Something like

    resolvedPaths = [
        ...(valueWithContext.resolvedValue as Map).values,
    ]

Not sure if there is a type error there, or if it actually would format nicer...

Reply:
Ah yes, spread format is win here, good call